### ISSUE TITLE
Performance issue / Long loading time

### ISSUE THREAD

**Issue by rygarok** [dunglas/symfony-docker#553]
Hi everyone.

I been trying to setup a new "template" for PHP 7.0 using docker and since Symfony recommended this project in the official documentation, I choose to go with it.

After setting up everything, I created a simple controller for the homepage that returns the base.html.twig just to test out the speed.

The first loading after first opening docker & the project took about 20 seconds (Okay).

Subsequent loading times (if I simply hit F5 to refresh the page) are always 4-5+ seconds. They all take extremely long to load a simple page on a basic / empty project. I tried multiple browsers and incognito mode as well just in case something would be interfering.

If I open the same project via wamp or xamp for example, I get much better speed (Around 400-500 ms)

![image](https://github.com/dunglas/symfony-docker/assets/45674551/3d821d06-62e8-4fd1-82b2-e03a771dbfc4)

My hardware is totally fine and CPU is never above 65%. Ram is good as well (about 70%).

Is there a reason why the basic installation of this project is so slow? What can be done to make it faster in dev?

Thank you

**Comment by HasteTheDay** [comment]
I was also giving FrankenPHP a go and was experiencing a similar performance issue.

The following fixed it for me:

Go to compose.override.yaml and follow the following comment instruction there:

> If you develop on Mac or Windows you can remove the vendor/ directory from the bind-mount for better performance by enabling the next line:

**Comment by maxhelias** [comment]
Yes, I couldn't suggest anything better than what @HasteTheDay suggests.

The performance issues are not related to the project but to the docker file system on the different OS.
Window is known for having the worst performance and we can't do anything about it on our own.
Thank you for your understanding.

**Comment by rygarok** [comment]
> I was also giving FrankenPHP a go and was experiencing a similar performance issue.
> 
> The following fixed it for me:
> 
> Go to compose.override.yaml and follow the following comment instruction there:
> 
> > If you develop on Mac or Windows you can remove the vendor/ directory from the bind-mount for better performance by enabling the next line:

Thanks a lot ! 

That did the trick; Everything load MUCH faster now (back to 200ms loading time).
